pkg/cfaws/assumer_aws_sso.go

```diff
@@ -141,6 +151,8 @@ func (c *Profile) SSOLogin(ctx context.Context, configOpts ConfigOpts) (aws.Cred
 func SSODeviceCodeFlow(ctx context.Context, cfg aws.Config, rootProfile *Profile) (*SSOToken, error) {
 	ssooidcClient := ssooidc.NewFromConfig(cfg)
 
+	shouldSilentStdout := ctx.Value("shouldSilentStdout")
```

Comment:
I think there is a better way to pass value to context. Suggestions appreciated.